client/core/trade.go

```diff
@@ -1845,9 +1845,13 @@ func (c *Core) redeemMatchGroup(t *trackedTrade, matches []*matchTracker, errs *
 
 	// Don't use (*Core).feeSuggestion here, since can incur an RPC request.
 	// If we don't have a synced book, use t.redemption
-	feeSuggestion := t.dc.bestBookFeeSuggestion(t.wallets.toAsset.ID)
+	// t.redeemFeeSuggestion is updated every tick and uses a rate directly
+	// from our wallet, if available. Only go looking for one if we don't have
+	// one cached.
+	feeSuggestion := atomic.LoadUint64(&t.redeemFeeSuggestion)
 	if feeSuggestion == 0 {
-		feeSuggestion = atomic.LoadUint64(&t.redeemFeeSuggestion)
+		// No new RPC requests.
```

Comment:
Can remove the comment.  "new" I assume means since before this diff.  If so, I get it.